FAQ/fr.md

```diff
@@ -0,0 +1,127 @@
+
+### J'ai une proposition de fonctionnalité, où puis-je la poster?
+
+Comme pour les signalements de bugs, les propositions de fonctionnalité doivent être envoyées sur [GitHub issues](https://github.com/Quaver/Quaver/issues). Choisissez simmplement “Feature Request” quand vous créez la publication!
+
+### Comment puis-je réduire latence audio des hitsounds sur Linux?
+
+Sur les options Audio vous pouvez trouver les paramètres  “Audio Device Period” et “Audio Device Buffer Length”. La rêgle de base est: tant que vous baissez ces valeurs la latence baisse, mais l'utilisation du CPU et la probabilité de problèmes de son augmentent. Essayez de paramètrer la "Period" par tranche de 2 ms et la "Buffer Length" à 8 ms et relancez le jeu. S'il y a des problèmes de son, essayez d'aumenter la "Period" ou la "Buffer Length".Pour réduire les problèmes de son nous recommendons d'ouvrir le fichier `/etc/security/limits.conf` et d'y ajouter une ligne comme ceci:
+```
+votre_nom_d'utilisateur      -   rtprio      99
+```
+Remplacez `votre_nom_d'utilisateur` par votre nom d'utilisateur Linux. Redemarrez votre système pour que ces changements prennent effet.Si vous avez de latence augmentant pendant que vous jouez, ouvrez `/etc/pulse/default.pa`, trouvez une ligne où il est écrit:
+```
+load-module module-udev-detect
+```
+et changez la en:
+```
+load-module module-udev-detect fixed_latency_range=yes
+```Et redemarrer le système. Notez que cela peut entraîner de sevères problèmes de son sur d'autres applications (Par exemple la discussion vocale de Discord).
+
+## Dépannage### Le jeu ne se lance pasRegardez bien si Steam est en train de tourner, car Quaver a besoin de Steam pour se lancer. Cela s'applique aussi à la version hors-ligne!
+
+#### Linux
+
+Quaver à besoin de `libgdiplus.so` et `libdl.so` d'installés pour se lancer. Si vous utilisez Debian, Ubuntu, Mint ou d'autres distributions basée sur Debian, essayez cette commande:
+```shell
+sudo apt install libc6-dev libgdiplus
+```Si vous utilisez Fedora, essayez cela:
+```shell
+sudo dnf install glibc-devel libgdiplus
+```Si vous utilisez Linux Arch, essayez cela:
+```shell
+sudo pacman -S glibc libgdiplus
+```
+
+#### Windows 7
+
+Pour lancer .NET Core sur Windows 7, vous avez besoin de dépendances supplémentaires.
+
+Suivez les étapes dans la boite violette sur [ce site](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites?tabs=netcore2x#net-core-dependencies).
+
+### Je ne peux pas importer mon mapset en utlisant
+
+ l'éditeurSi votre mapset ne peut pas s'importer avec le client, vous pouvez l'importer manuellement sur la [Importation de Mapset](https://quavergame.com/upload/mapset/).
```

Comment:
_l'éditeur_ should be in the title

vous pouvez l'importer manuellement sur la **page** [Importation de Mapset](https://quavergame.com/upload/mapset/).

Reply:
I don't know how this one occured because it should have been like the others but it's fixed now
( You made me aware of the "utlisant" btw haha )